docs/Configuration.md

```diff
@@ -234,6 +234,7 @@ Here is a list of all features which can be toggled on/off and their default val
 | `OptimizeTransfers`                        | OTP will inspect all itineraries found and optimize where (which stops) the transfer will happen. Waiting time, priority and guaranteed transfers are taken into account.                                 |         ✓️         |         |
 | `ParallelRouting`                          | Enable performing parts of the trip planning in parallel.                                                                                                                                                 |                    |         |
 | `TransferConstraints`                      | Enforce transfers to happen according to the _transfers.txt_(GTFS) and Interchanges(NeTEx). Turing this _off_ will increase the routing performance a little.                                             |         ✓️         |         |
+| `TransmodelGraphQlApi`                     | Enable Transmodel(NeTEx) GraphQL API.                                                                                                                                                                     |         ✓️         |    ✓️   |
```

Comment:
```suggestion
| `TransmodelGraphQlApi`                     | Enable Transmodel (NeTEx) GraphQL API.                                                                                                                                                                     |         ✓️         |    ✓️   |
```
It's difficult to find a definitive source for spacing rules in the English language, but the Internet seems to be in agreement, for example:
https://english.stackexchange.com/questions/241408/do-parentheses-need-spaces-either-side